server/controllers/getUserId.js

```diff
@@ -0,0 +1,24 @@
+const { SECRET } = process.env
+const jwt = require('jsonwebtoken')
+const cookie = require('cookie')
+
+const checkUser = (req, res) => {
+  if (req.headers.cookie) {
+    const { token } = cookie.parse(req.headers.cookie)
+    jwt.verify(token, SECRET, (err, result) => {
+      if (err) {
+        return res.status(400)
+      }
+      if (result) {
+        const id = result.id
+        const name = result.name
+        const role = result.role
+
+        return res.json({ success: true, id, name, role })
+      }
+    }
+    )
+  } else return res.json({ success: false, message: 'faild to get user' })
```

Comment:
no cookie is a 401 unauthorized.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401

Reply:
we will handle all the err after merging